runtime/doc/various.txt

```diff
@@ -557,5 +557,34 @@ LessInitFunc in your vimrc, for example: >
 	  set nocursorcolumn nocursorline
 	endfunc
 <
+==============================================================================
+3. Commenting							*commenting*
+
+Nvim supports commenting and uncommenting per line. It is done by toggling
+the status of consecutive lines: if all lines are commented - uncomment them,
+comment otherwise.
+
+Comment structure is directly inferred from 'commentstring' at cursor: text
+before and after `%s` is taken as left and right part of comment.
+
+Commenting line is done by inserting left comment part at appropriate indent
+and right part at line's end.
+Uncommenting is done by removing left and right parts from start and end of
+line (ignoring whitespace).
+Blank lines are also commented and uncommented with trimmed versions of
+comment parts for a more intuitive experience.
+
+							*gc*
+gc{motion}		Toggle lines over {motion}.
```

Comment:
```suggestion
gc{motion}		Toggle comment for lines over {motion}.
```
(and similarly below).

Reply:
One reason I did not use that is because "Toggle comment for lines" does not sound as good wording for new users. It can be "Toggle commented status of lines", but this is too much. So decided to leave at just "Toggle" which is described in the first line.